src/annotator-tools/AnnotationTool/AnnotationTool.ts

```diff
@@ -18,15 +18,10 @@ import {
 } from "utils/annotator";
 
 export abstract class AnnotationTool extends Tool {
-  /**
-   * RoiManager of the annotated image.
-   * https://image-js.github.io/image-js/#roi
-   */
-  manager: ImageJS.RoiManager;
```

Comment:
Deleted because it's never uses. Other tools use ROIManager, but  they create it themselves.